lib/client.ts

```diff
@@ -36,14 +36,6 @@ const notNil = compose(
   isNil
 )
 
-function authValid(response): any {
-  if (response.status === 403 || response.status === 401 || !response.ok) {
-    throw new Error(`${response.status} ${response.statusText}`)
```

Comment:
This was swallowing any invalid response, not allowing us to show these errors to the user.